src/app/pages/registration/register-form/register-form.component.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RegisterFormComponent } from './register-form.component';
+import { DebugElement } from '@angular/core';
+import { BrowserModule, By } from '@angular/platform-browser';
+import { FormBuilder, FormGroup, Validators, NgForm } from '@angular/forms';
+import { ReactiveFormsModule, FormsModule } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { HttpClientModule } from '@angular/common/http';
+
+describe('RegisterFormComponent', () => {
+  let component: RegisterFormComponent;
+  let fixture: ComponentFixture<RegisterFormComponent>;
+  let de: DebugElement;
+  let el: DebugElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ RegisterFormComponent ],
+      imports: [
+        ReactiveFormsModule,
+        FormsModule,
+        HttpClientModule,
+        BrowserAnimationsModule]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RegisterFormComponent);
+    component = fixture.componentInstance;
+    de = fixture.debugElement.query(By.css('form'));
+    el = de.nativeElement;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('trigger onsubmit method', async(() => {
```

Comment:
@akram256, it's customary that test descriptions begin with should and such even a non-programmer can read it from the 'it' keyword and know what exactly it tests.
This can be better written as `it('should trigger the onSubmit method',....`